Makefile.common

```diff
@@ -147,14 +147,16 @@ LINKCMX:=$(CORECMA:.cma=.cmxa) $(STATICPLUGINS:.cmo=.cmx)
 
 ## we now retrieve the names of .vo file to compile in */vo.itarget files
 
-THEORIESVO:= $(foreach f, $(wildcard theories/*/vo.itarget), \
-                $(addprefix $(dir $(f)),$(shell cat $(f))))
+GENVOFILES := $(GENVFILES:.v=.vo)
 
-PLUGINSVO:=  $(foreach f, $(wildcard plugins/*/vo.itarget), \
-                $(addprefix $(dir $(f)),$(shell cat $(f))))
+THEORIESVO := $(patsubst %.v,%.vo,$(shell find theories -type f -name "*.v")) \
+              $(filter theories/%, $(GENVOFILES))
 
-ALLVO:= $(THEORIESVO) $(PLUGINSVO)
-VFILES:= $(ALLVO:.vo=.v)
+PLUGINSVO := $(patsubst %.v,%.vo,$(shell find plugins -type f -name "*.v" | grep -v plugins/micromega/MExtraction.v)) \
```

Comment:
Hmm, isn't [this](https://github.com/coq/coq/blob/trunk/plugins/micromega/MExtraction.v) it?